website/src/components/Visualisations/IIGOPayments/Util/ProcessPaymentsData.test.ts

```diff
@@ -0,0 +1,2163 @@
+import { processPaymentsData } from './ProcessedPaymentsData'
+
+const testInput = {
```

Comment:
@Darrekt do we already have some sample test inputs?

Reply:
@Darrekt used some thing similar in his testing, but there is a lot of wrong/weird values inside so I change a few things in mine :(

Reply:
ah, that's fine then, it's just a bit yuuge